test/test_meta.py

```diff
@@ -202,7 +223,7 @@ def should_check_strides(func):
     if any(r.alias_info.before_set for r in func._schema.returns if r.alias_info):
         return True
     # TODO: check for TensorIterator
-    return False
+    return True
```

Comment:
@ezyang, @eellison  

We should enable stride check for all ops ?

Reply:
We should, independently of the rest of the changes.